runbotics/runbotics-actions-windows/src/automation/office/excel.action-handler.ts

```diff
@@ -91,6 +97,27 @@ export default class ExcelActionHandler extends StatefulActionHandler {
                 .Value();
     }
 
+    async getCells(
+        input: ExcelGetCellsActionInput
+    ): Promise<Record<string, any>> {
+        const optionalWorksheet = input?.worksheet;
+        const openedWorksheet = this.session.ActiveSheet.name;
+        const cellValues: Record<string, any> = {};
```

Comment:
any to unknow

Reply:
and in translations change obj format to JSON obj 

+ handle wrong input -> add  

 private checkIsObject(value: unknown): void {
        if (typeof value !== 'object') {
            throw new Error(`Target Excel structure must be variable of a JSON object e.g.:
            {
                "A1": "value",
                "B3": "another value"
            }
            Check targetExcelStructure in Input tab above.`);
        }
    }